releng/org.eclipse.n4js.targetplatform/N4JS.setup

```diff
@@ -177,11 +177,11 @@
   <setupTask
       xsi:type="setup:VariableTask"
       name="n4js.target.platform"
-      defaultValue="N4JS_2018_09"
-      storageURI="scope://Workspace">
+      defaultValue="N4JS_2019_03"
+      storageURI="scope://Workspace">    <!-- label! -->
```

Comment:
label comments are probably obsolete?

Reply:
This is referred to in the documentation. It's used to find places in the setup files, etc. that need updating when bumping Eclipse, Xtext, etc. and are difficult to find, otherwise.